x-pack/plugins/spaces/server/lib/get_spaces_usage_collector.ts

```diff
@@ -38,14 +39,14 @@ interface SpacesAggregationResponse {
 async function getSpacesUsage(
   callCluster: CallCluster,
   kibanaIndex: string,
```

Comment:
__optional nit:__ jsdoc for this method declares some `server`, but not `kibanaIndex`.